### DISCUSSION TITLE
Welcome to FrivReborn Discussions!

### DISCUSSION THREAD

**Discussion by Tsyuay** [Tsyuay/FrivReborn#10] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by frivis** [comment]
hello when i go to the site it says this:
Whoops!

404.

Babs couldn't find that page.

Seems like you are a bit lost.

You are lucky! you can go back to the main page.

**what should I do?????**

**Comment by Tsyuay** [comment]
Hi, that's because some games are still missing in Friv Classic Reborn, but since I haven't worked in this project in so long I'm planning to maybe continue it.

Regards, Panda Fake Projects

**Reply by kpro199procreator** [reply]
Hello Tsyuay, I would like to help you complete all the games in the friv library.
